packages/server/src/test/java/software/uncharted/terarium/hmiserver/configuration/TestReBACService.java

```diff
@@ -25,7 +25,7 @@
 import software.uncharted.terarium.hmiserver.utils.rebac.askem.RebacPermissionRelationship;
 
 @Service
-@Primary
+@Profile("test")
```

Comment:
There are a number of broken tests right now - what was this change for?

Reply:
Ah it was a temporary fix to work around the issue that test functions are intercepting real functions in the run time and breaks some functionality on the app when it's ran using VSCode. I'll revert it back. And let @kbirk do proper fix :pray: